tests/common/mellanox_data.py

```diff
@@ -17,6 +17,15 @@
         "psus": {
             "number": 2,
             "hot_swappable": True
+        },
+        "cpu_pack": {
+            "number": 1
+        },
+        "cpu_cores": {
+            "number": 2
+        },
+        "ports":{
```

Comment:
i am confused on the way number of ports is calculated.
SN2100 is a 16 ports switch unless you breakout the cables, so this is configuration based.
i think you should not have it as hard coded value but instead read the information from the configuration to have the right number.

Reply:
correct the typo of ports number on 2100, this mellanox_data is designed to put never changed platform related info, and check this info against the facts on the DUT, so I think hardcode is OK.

Reply:
Is it part of the test that you verify it against the DUT? What will happen if the SKU is with split ports?

Reply:
No issue with split ports, check SFP symbolink tests and SFP presence test used this new added ported number, they all work well on the testbeds with split ports.